package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "nutshell-alces",
+  "version": "1.0.0",
+  "description": "PanAm Airlines",
+  "main": "src/javascripts/main.js",
+  "scripts": {
+    "start": "webpack-dev-server --mode development --open",
+    "build": "webpack --mode production --module-bind js=babel-loader",
+    "deploy": "npm run build && firebase deploy"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nss-evening-cohort-12/nutshell-alces.git"
+  },
+  "keywords": [],
+  "author": "Phonesalo Sihanorak",
```

Comment:
you could put a comma separated list in here if you wanted to

Reply:
Of course.....can't leave the other two out!! LOL